src/basic/time-util.c

```diff
@@ -560,7 +560,7 @@ int dual_timestamp_deserialize(const char *value, dual_timestamp *t) {
         assert(t);
 
         if (sscanf(value, "%" PRIu64 "%" PRIu64, &a, &b) != 2) {
-                log_debug("Failed to parse dual timestamp value \"%s\": %m", value);
+                log_debug_errno(errno, "Failed to parse dual timestamp value \"%s\": %m", value);
```

Comment:
this doesn't look right. If I read the sscanf() man page right, errno is not set if sscanf returns positive or zero (because reading anything successfully even if we don't read the full number of items is considered success). Hence, the above error message will be printed on actual errors and in the case when we read a short number of items, but in the latter we'll still output %m and we really shouldn't---